lambdas/user-list-search-corpus-parser-hydration/src/bulkIndex.ts

```diff
@@ -0,0 +1,72 @@
+import { config } from './config';
+import * as Sentry from '@sentry/aws-serverless';
+import { serverLogger } from '@pocket-tools/ts-logger';
+import { BulkRequestPayload } from './types';
+import fetchRetry from 'fetch-retry';
+const newFetch = fetchRetry(fetch);
+/**
+ * Make an HTTP request to the bulk index of elasticsearch to
+ * index documents in corpora depending on language.
+ * @param record SQSRecord containing forwarded event from eventbridge
+ * @returns the batch response with identifiers if any events failed
+ */
+export async function bulkIndex(
+  records: BulkRequestPayload[],
+): Promise<string[]> {
+  const failedMessageIds: string[] = [];
+  // The new elasticsearch client doesn't work on AWS
```

Comment:
Doesnt the new one work since our upgrade?

Reply:
oh, yeah I just copied since I already had it 😅

Reply:
the request from the new client is identical to this request, just in object form. Since I already have it working from before and it's the only ES request, I'm going to keep it unless you feel strongly

Reply:
nah fine by me just wanted to ask